.travis.yml

```diff
@@ -147,6 +147,16 @@ jobs:
       go: $TRAVIS_GO_VERSION
       stage: test
 
+    # Agent
```

Comment:
Do we have this also in our jenkins build?

Reply:
yes https://beats-ci.elastic.co/job/elastic+beats+pull-request+multijob-linux/6636/beat=x-pack%2Fagent,label=linux-immutable/